tests/test_issue_trackers.py

```
@@ -1,5 +1,5 @@
 from gito.issue_trackers import extract_issue_key
-
+import gito
```

Ruff (0.11.9)

`gito` imported but unused

Remove unused import: `gito`

(F401)